modules/system/classes/asset/PackageManager.php

```diff
@@ -24,10 +24,14 @@ class PackageManager
 {
     use \Winter\Storm\Support\Traits\Singleton;
 
+    public const TYPE_THEME = 'theme';
+    public const TYPE_MODULE = 'module';
+    public const TYPE_PLUGIN = 'plugin';
```

Comment:
At some point it might be worth making enums for the extension types in Winter to centralize this sort of thing (and others like paths etc)